quickstarts/browser-segment-investigation/config.yml

```diff
@@ -0,0 +1,38 @@
+# Name of the quickstart (required)
+name: browser-segment-investigation
+
+# Displayed in the UI (required)
+title: Browser Segment Investigation Quickstart 
+
+# Long-form description of the quickstart (required)
+description: |
+ Review the extent of crushed urls so you know which urls you need to add segments for in the browser segment allow list.
```

Comment:
Google results' definition of truncated urls is different from the browser configuration issue that this dashboard is meant to address.  
Truncated urls are necessary and desirable in browser.  Crushed urls impact the usefulness of the platform.  I put it to browser wizards to weigh in.   I'll circle back to this tomorrow.  Thanks for raising it.
@jpvajda

Reply:
@khickey-newrelic thanks for confirming that term, if it's recognized in the industry, it can definitely be used, I just never heard it before and couldn't find any sources on it, apart from our own docs page.